HACKING.md

```diff
@@ -140,7 +140,6 @@ like `fedora-coreos-devel` if you'd like.
 
 ```
 oc new-project fedora-coreos
-oc delete project myproject
```

Comment:
The default `oc cluster up` behaviour starts you up with a project called `myproject`, right?
Oh or are you saying there's no need to go out of our way to delete it?

Reply:
yeah mostly you don't need to go out of your way to delete it. It's also not really necessary and may even be destructive if you target an existing cluster rather than going the `oc cluster up` route.